File: src/main/java/org/example/bookfd/domain/School.java
package org.example.bookfd.domain;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "school")
public class School {

    @Id
    private int id;
    private String name;
    private String nameCity;
    private int numClass;
    private Boolean endSchool;

    @Override
    public String toString() {
        return "School{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", nameCity='" + nameCity + '\'' +
                ", numClass=" + numClass +
                ", endSchool=" + endSchool +
                '}';
    }
}

File: src/main/java/org/example/bookfd/domain/School.kt
package org.example.bookfd.domain

import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "school")
class School {
    @Id
    var id = 0
    var name: String? = null
    var nameCity: String? = null
    var numClass = 0
    var endSchool: Boolean? = null

    override fun toString(): String {
        return "School{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", nameCity='" + nameCity + '\'' +
                ", numClass=" + numClass +
                ", endSchool=" + endSchool +
                '}'
    }
}
